Code/GraphMol/SmilesParse/SmilesParse.cpp

```diff
@@ -421,6 +437,13 @@ RWMol *SmilesToMol(const std::string &smiles,
       delete res;
       throw;
     }
+    if (res->hasProp(SmilesParseOps::detail::_needsDetectBondStereo)) {
+      // we encountered either wiggly bond in the CXSMILES,
```

Comment:
This is interesting! I know that in mol files, the spec says that stereo comes from the coordinates for 3d, and the bond directions if 2d. And the atom parities are meaningless...

It seems like you've updated CXSMILES to follow this same logic. I haven't seen anything in the CX docs specifying which of the 3 mechanisms is canonical. It would be worth documenting (perhaps just in the commit message or PR discussion) if you found some source for this.


My one experiment with marvinjs agrees with you:

    CC1CCC[C@@H]1O |wU:1.0,(0.5678,1.9393,;0.5678,1.1143,;1.2352,0.6294,;0.9803,-0.1553,;0.1553,-0.1553,;-0.0997,0.6294,;-0.8843,0.8843,)|

Seems to get stereo on the extracyclic CC bond, but not the CO bond. And the atom marked with @ is actually given ? stereo.

Reply:
yeah, would be worth documenting this stuff

Reply:
I had noticed the same thing internally so I think this is how it works as well.